code/modules/mob/living/carbon/alien/larva/larva.dm

```diff
@@ -26,6 +26,10 @@
 
 	AddAbility(new/obj/effect/proc_holder/alien/hide())
 	AddAbility(new/obj/effect/proc_holder/alien/larva_evolve())
+
+	for (var/mob/living/carbon/alien/a in aliens)
+		a << "<span class='alertalien'>[src] has been born at [src.loc.loc ? src.loc.loc : src.loc ? src.loc : "the void"]!</span>"
```

Comment:
no really,isn't there a get area proc,i'm 99% sure there is,this is just ugly and not precise